packages/postgres-database/src/simcore_postgres_database/models/workspaces.py

```diff
@@ -0,0 +1,76 @@
+import sqlalchemy as sa
+
+from ._common import column_created_datetime, column_modified_datetime
+from .base import metadata
+
+workspaces = sa.Table(
+    "workspaces",
+    metadata,
+    sa.Column(
+        "workspace_id",
+        sa.BigInteger,
+        nullable=False,
+        autoincrement=True,
+        primary_key=True,
+        doc="Workspace index",
+    ),
+    sa.Column("name", sa.String, nullable=False, doc="Display name"),
+    sa.Column("description", sa.String, nullable=True, doc="Short description"),
```

Comment:
MINOR: it you intend this to be short, make sure to serialize from a model string that truncates the size